src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -1194,7 +1226,7 @@ public function testSubmitSingleExpandedNonRequiredFalse()
         $form->submit(false);
 
         $this->assertNull($form->getData());
-        $this->assertNull($form->getViewData());
+        $this->assertEmpty($form->getViewData());
```

Comment:
@webmozart "fixing" those tests seems logic to me as the placeholder is an empty string.

And it is coherent with this [comment](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php#L329).